docs/sink/flink-connector-oceanbase.md

```diff
@@ -148,31 +148,33 @@ CREATE TABLE t_sink
+| direct-load.timeout            | No                    | No                     | 7d                       | Duration | Timeout for direct load task.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| direct-load.heartbeat-timeout  | No                    | No                     | 60s                      | Duration | Client heartbeat timeout in direct load task.                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| direct-load.heartbeat-interval | No                    | No                     | 10s                      | Duration | Client heartbeat interval in direct load task.                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| direct-load.load-method        | No                    | No                     | full                     | String   | Direct load load mode: full, inc, inc_replace. full: full direct load, default value. inc: normal incremental direct load, primary key conflict check will be performed, observer-4.3.2 and above support, dupAction REPLACE is not supported for the time being.inc_replace: special replace mode incremental direct load, no primary key conflict check will be performed, directly overwrite the old data (equivalent to the effect of replace), dupAction parameter will be ignored, observer-4.3.2 and above support. |
```

Comment:
You can use some html tags like `<li>` and `<code>` to rearrange the description.

Reply:
Converted to html format.